test/collection_test.js

```diff
@@ -210,49 +209,47 @@ describe("Collection", () => {
       sandbox.stub(client, "execute").returns(Promise.resolve({data: 1}));
     });
 
+    it("should throw if record is not an object", () => {
+      expect(() => coll.updateRecord(2)).to.Throw(Error, /required/);
+    });
+
+    it("should throw if id is missing", () => {
+      expect(() => coll.updateRecord({})).to.Throw(Error, /required/);
```

Comment:
Could we have more precise test strings to reflect we're actually getting the expected messages?